### PR TITLE
Address review feedback + other minor changes

### PR DESCRIPTION
**Pull request by adrianwong**
The "other minor changes" are:

1. Handle additional command line options that don't require values.
2. Consistent use of American English. `rasterise` -> `rasterize`, as we already spell "colour" as "color".
3. Make `rasterPages` singular, as we only really allow a single page number to be specified anyway.

### CODE REVIEW

File: src/main/java/com/princexml/wrapper/AbstractPrince.java [hunk 1]
```diff
@@ -173,8 +173,8 @@ protected boolean readMessages(BufferedReader reader) throws IOException {
 
                 if (events != null && msgTag.equals("msg")) {
                     handleMessage(msgBody);
-                } else if (msgTag.equals("dat")) {
-                    // TODO Data messages?
+                } else if (events != null && msgTag.equals("dat")) {
+                    handleDataMessage(msgBody);
```
Comment (mikeday) [inline src/main/java/com/princexml/wrapper/AbstractPrince.java:177]: Perhaps slightly clearer to test events on a nested if, just so the outer if is basically switching on msgTag.

**Review by mikeday — APPROVED** [review]
Looks good!

File: src/main/java/com/princexml/wrapper/AbstractPrince.java (hunk 1 above)
Comment (adrianwong) [reply src/main/java/com/princexml/wrapper/AbstractPrince.java]: I've nested the `events` check and used a switch for the `msgTag` check.

**Review by mikeday — APPROVED** [review]
That looks fine.

**Pull request merged**